src/chartkit/components/Legend/Legend.js

```diff
@@ -33,12 +33,16 @@ const Legend = ({ legends = [], direction = DIRECTION_ROW, theme }) => {
     <div style={style}>
       <svg height={maxHeight} xmlns="http://www.w3.org/2000/svg">
         {legends.map((l, i) => {
-          let xOffset = 0;
-
-          xOffset = itemWidth + itemsSpacing;
+          let xOffset = itemWidth + itemsSpacing;
```

Comment:
I'm still learning to grow up and get over my anxiety when seeing a `let` over `const`